test/github-issues/8667/issue-8667.ts

```diff
@@ -0,0 +1,85 @@
+import "reflect-metadata"
+import { expect } from "chai"
+import { DataSource } from "../../../src"
+import { Post } from "./entity/Post"
+import {
+    createTestingConnections,
+    reloadTestingDatabases,
+    closeTestingConnections,
+} from "../../utils/test-utils"
+
+describe("github issues > #8667 Support RETURNING clause in SQLite", () => {
+    let connections: DataSource[]
+    let post: Post
+
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Post],
+            schemaCreate: true,
+            dropSchema: true,
+            enabledDrivers: ["sqlite", "better-sqlite3", "sqljs"],
+        })
+
+        post = new Post()
+        post.title = "title"
+        post.text = "text"
+    })
+
+    beforeEach(() => reloadTestingDatabases(connections))
+    after(() => closeTestingConnections(connections))
+
+    it("should return all columns via QueryRunner", async () =>
```

Comment:
Would be nice if you could update `test/functional/query-builder/returning/query-builder-returning.ts` - you can simply add the drivers and update the expected queries for SQLite-like drivers.